src/openms/include/OpenMS/CHEMISTRY/IsotopeDistribution.h

```diff
@@ -155,6 +155,78 @@ namespace OpenMS
     */
     void estimateFromWeightAndComp(double average_weight, double C, double H, double N, double O, double S, double P);
 
+
+    /**
+        @brief Estimate peptide fragment IsotopeDistribution from the precursor's average weight,
+        fragment's average weight, and a list of isolated precursor isotopes.
```

Comment:
I have no strong opinion on this. I merely asked out of curiosity. 
More important in my view would be to document that 0 corresponds to the mono-isotopic peak.